cpp/tests/io/parquet_test.cpp

```diff
@@ -1087,7 +1087,7 @@ TEST_F(ParquetWriterTest, MultiIndex)
   cudf::test::expect_metadata_equal(expected_metadata, result.metadata);
 }
 
-TEST_F(ParquetWriterTest, HostBuffer)
+TEST_F(ParquetWriterTest, BufferSource)
```

Comment:
I can also leave the current test as it is and add another `DeviceBuffer` test case. The main reason to update existing tests is to minimize duplicates.